pagopa/io-app
# chore: [IOBP-893] Adapted new IDPay amount with cents

## Short description
This PR adapts the amount using cents value;

## List of changes proposed in this pull request
- Replaced the `formatNumberAmount` utility function with the `formatNumberCentsToAmount` in order to format the amount considering the cents

## How to test
- Enable the `pagoPA` test env. with the production .env;
- Enable the `IDPay` feature flag;
- Open the Wallet home page and choose an initiative;
- Check that the initiative amount text is divided by 100 showing the right decimal amount;

## Preview
|Before|After|
|-|-|
|<img src="https://github.com/user-attachments/assets/b9c2ed41-d7c2-48a0-9952-ecccafeae6b7" width="300px" />|<img src="https://github.com/user-attachments/assets/0c4024fe-ecf6-49eb-8792-71687eb702e7" width="300px" />
